components/TokenManagePage/Media.tsx

```diff
@@ -51,6 +59,15 @@ const Media = () => {
               placeholder="enter a description"
             />
           </div>
+
+          <button
+            className="bg-black text-grey-eggshell w-fit px-8 py-2 rounded-md disabled:opacity-50"
+            onClick={setMetadata}
+            disabled={isLoading || isSaving || !isOwner}
+          >
+            {isSaving ? "saving..." : "Save"}
+          </button>
+          {!isOwner && <OwnerWarning />}
```

Comment:
move the `isOwner` check within the `<OwnerWarning />` component.

Question: Is there ever a case when you'd want to show the `<OwnerWarning />` component when `isOwner` resolves to `true`?
- If `no`, why not move the `isOwner` check within the `<OwnerWarning />` component and return `null` if `isOwner` resolves to `true`?